internal/builder/v1alpha1/task.go

```diff
@@ -22,7 +22,6 @@ import (
 	"github.com/tektoncd/pipeline/pkg/apis/config"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
-	"github.com/tektoncd/pipeline/pkg/reconciler/taskrun/resources"
```

Comment:
We should *never* import `pkg/reconciler` packages outside of `pkg/reconciler` :smiling_imp:

Reply:
💯